test/jdk/java/awt/print/PageFormat/SetOrient.java

```diff
@@ -54,23 +106,4 @@ public void init() {
             throw new RuntimeException(e.getMessage());
```

Comment:
In other PRs, I guess we do
```

 ex.printStackTrace();
                    String msg = "PrinterException: " + ex.getMessage();
                    PassFailJFrame.forceFail(msg);
```

Reply:
You're right. Changed to align with other PRs.